src/MCPClient/lib/clientScripts/validateFile.py

```diff
@@ -153,14 +153,18 @@ def _execute_rule_command(self, rule):
         # NOTE: this requires that the stdout of all validation FPR commands be
         # a dict (preferably a JSON object) with an ``eventOutcomeInformation``
         # boolean attribute.
-        if output.get('eventOutcomeInformation') != 'pass':
-            print('Command {} indicated failure with this output:\n\n{}'.format(
-                rule.command.description, stdout), file=sys.stderr)
+        if output.get('eventOutcomeInformation') == 'pass':
+            print('Command "{}" was successful'.format(rule.command.description))
+        else:
+            print('Command "{}" indicated failure with this'
```

Comment:
When using multiple parenthesis, do you think it's a good idea to number them, it might be clearer? e.g. from below there are three we could number: `print('Creating {0} event for {1} ({2})'.format(self.purpose, self.file_path, self.file_uuid))`

Reply:
I think maybe [named placeholders](https://pyformat.info/#named_placeholders) is even clearer, so I've changed all instances of `.format` with multiple vars to use that.